tests/main/snapshot-users/task.yaml

```diff
@@ -0,0 +1,122 @@
+summary: Check that the basic snapshots functionality works for different users
+
+prepare: |
+    snap install test-snapd-tools
+
+execute: |
+    # use the snaps, so they create the dirs
+    test-snapd-tools.echo
+    su -p -c "/snap/bin/test-snapd-tools.echo" test
```

Comment:
Fedora and Arch do not have `/snap` by default:
```
#shellcheck source=tests/lib/dirs.sh
. "$TESTSLIB"/dirs.sh
su -p -c "$SNAP_MOUNT_DIR/bin/test-snapd-tools.echo" test
```

Reply:
Can we just use `snap run test-snapd-tools.echo` instead?

Reply:
This fails on:
```
/bin/bash: line 61: SNAP_MOUNT_DIR: unbound variable
```
We probably need to source dirs.sh

Reply:
done